### ISSUE TITLE
protein sequence output file

### ISSUE THREAD

**Issue by graciegracekk** [linnabrown/run_dbcan#123]
I'm running the dbcan on faa files on linux to predict the CAZymes and saw that I can get a protein sequence of the gene (this link shows an output of the gene with the protein sequence: https://bcb.unl.edu/dbCAN2/data/blast/20221121163906/CAZyme.pep) . How do I get that ^^output file?

**Comment by linnabrown** [comment]
Sorry I could not get your point. Could you rephrase ur question?

**Comment by graciegracekk** [comment]
hi,

1. I want to get a protein sequence of the CAZyme when I run the dbCAN.
How do I get the output file that looks like this? (has the following info)

 >NC_000913.3_1054 3.2.1.-:GH73(160-295):GH73_e114:GH73:N:3

MISDSKLLASAAWDAQSLNELKAKAGEDPAANIRPVARQVEGMFVQMMLKSMRDALPKDG
LFSSEHTRLYTSMYDQQIAQQMTAGKGLGLAEMMVKQMTPEQPLPEESTPAAPMKFPLET
VVRYQNQALSQLVQKAVPRNYDDSLPGDSKAFLAQLSLPAQLASQQSGVPHHLILAQAAL
ESGWGQRQIRRENGEPSYNLFGVKASGNWKGPVTEITTTEYENGEAKKVKAKFRVYSSYL
EALSDYVGLLTRNPRYAAVTTAASAEQGAQALQDAGYATDPHYARKLTNMIQQMKSISDK
VSKTYSMNIDNLF*


2.Can I run dbCAN on both FAA and FNA files? or is it just FNA files?


Grace

**Comment by linnabrown** [comment]
This file located in the temporary folder

        self.input_folder = args.input if args.input.endswith("/") else args.input +"/"
2. You can run faa or fna. You don't need to put them at the same time.
